apps/api/src/app/notification-template/dto/create-notification-template.request.dto.ts

```diff
@@ -14,7 +14,7 @@ export class CreateNotificationTemplateRequestDto implements ICreateNotification
   @ApiProperty()
   @IsString()
   @IsDefined({
-    message: 'Notification group must be provided',
+    message: 'Notification group must be provided ',
```

Comment:
do we need that space?

Reply:
I found that if we won't have it that the next argument will miss a space, so I added it here but if it is persistent in all cases maybe looking at how to fix it globally will be a better idea.